### DISCUSSION TITLE
Any download work after update

### DISCUSSION THREAD

**Discussion by Djahmo** [aclap-dev/video-downloadhelper#663] [Help and Q&A]
### Your question

Wtf is this update break all the app. 

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://v5.voiranime.com/anime/boku-no-hero-academia-7/boku-no-hero-academia-7-04-vostfr/",
  "error": {
    "details": "SideDownload error: : [tcp @ 0000028a8f5ab9c0] Failed to resolve hostname 908472281.tapecontent.net: The name does not resolve for the supplied parameters\r\n[in#0 @ 0000028a90eeb580] Error opening input: I/O error\r\nError opening input file https://908472281.tapecontent.net/radosgw/rJZbWm9JVASb1d9/6T0wzXDpo3OeZQp1el9cumzDEOzzbmoJiQ_9vPokf_jXwqxK-xq9B9fb6zf6BLMOzYkcavgu7oLtmD_kUN_snfg9m0rpC9D5rakCl9bhU4LsccB4LEVGA1G1p_QxZNCxXVcXCzcjBEvaq62kNu4nwEoF-XTBivQhapG6Aczlrur28i3ALwPc7Q5iLyk0hzZVbjo_IKW3svmdPiFF835vEYSUAVGl7iZpnrODTBwcpLtDSnHZ-1vlPngNt8ai90GaENY4nCHKI-AxGfO1WijNp5QsyUjfKgsf_uQjwCEabtjZxuoLlc-M0UnPo2Q/Boku+no+Hero+Academia+7+-+04+VOSTFR.mp4.mp4?stream=1.\r\nError opening input files: I/O error\r\n",
    "downloadable_id": "downloadable_833242948607033",
    "error": "coapp_failure",
    "id": "coapp_failure_078c642d-b715-4460-bd46-60da278f5d0c"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.

Your problem report cites a URL in the page_url tag.  When I visited there, I had to satisfy a captcha, then dismiss a couple of ads.  Finally, playback of the video was possible, & at that point, VDH offered me this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/fddc0766-c2f2-4094-b9e4-5de84f99c4eb)

So I downloaded it.

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/3bb0dbf3-aa4f-4a0b-aeae-2ce291c82f31)

That's a pretty good download speed.  The file I got looked like this:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/9d101263-c000-4026-8e5e-0bcbc607b0de)

I had no intention of sitting through this, but just after I launched playback, this caption caught my eye:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/e44deacb-fd73-4cb4-af40-6713b8017dab)

I laughed at that.  Anyway, after that short chuckle, I just skimmed through the rest of this, sampling it at intervals.  That image shows the video was fine.  You'll have to take my word for it that the audio was just as fine.

I suggest you should be running the same level of VDH & the CoApp that I am.  I would also suggest that any problem report that starts with "wtf" probably deserves to be ignored.  But I'm in a good mood today.  In future, be less offensive & combative.  That's not a request.  Improve your manners.  I won't put up with such behavior again.

**Reply by GrampaWildWilly** [reply]
It's possible that this is another in the long history of examples of videos on which Firefox does a better job than Chrome.
